pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -27,10 +27,6 @@
 
   hoc2018_tutorial_mchoc_description: 'Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code.'
 
-  hoc2018_minecraft_aquatic_title1: "Minecraft Voyage Aquatic"
-  hoc2018_minecraft_with_microsoft: "In partnership with"
-  hoc2018_minecraft_thank_microsoft: "We thank Microsoft for supporting our vision and mission to ensure every child has the opportunity to learn computer science and the skills to succeed in the 21st century."
```

Comment:
have you verified that these strings aren't used anywhere else?

Reply:
Yes.  These strings are not being used anywhere else.  There is also another string "hoc2018_minecraft_aquatic_title2" which is not being used but it is unrelated to the change in this PR.  A follow-up PR will be opened to delete the string.